app/test_split_integer.py

```diff
@@ -2,20 +2,27 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    goals = split_integer(32, 6)
```

Comment:
According to the checklist, avoid creating unnecessary variables for simple tests. You can directly use `assert sum(split_integer(32, 6)) == 32`.